CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Adds support to build Docker-based Secrets Provider images on CyberArk NG Laptops. [cyberark/secrets-provider-for-k8s#424](https://github.com/cyberark/secrets-provider-for-k8s/pull/424) 
```

Comment:
This should probably be left out of the CHANGELOG.md, since it's not a user-facing change.

Reply:
ok, will remove.